tutorials/deploy-laravel-on-serverless-containers/index.mdx

```diff
@@ -33,7 +33,7 @@ Scaleway provides a seamless environment for running and managing containers, of
 </Message>
 
 ## Creating a SQS/SNS queue
-Laravel applications make use of [queues](https://laravel.com/docs/10.x/queues) to process long running jobs in the background. As this feature of the Laraval framework is nearly always used, we will configure it and leverage the [Messaging & Queuing](https://www.scaleway.com/en/messaging-and-queuing/) product from Scaleway. The [Scaleway Developer documentation](https://www.scaleway.com/en/docs/serverless/messaging/) provides clear information on how this managed service works and can be configured.
+Laravel applications make use of [queues](https://laravel.com/docs/10.x/queues) to process long running jobs in the background. As this feature of the Laraval framework is nearly always used, we will configure it and leverage the [Messaging & Queuing](https://www.scaleway.com/en/messaging-and-queuing/) product from Scaleway. The [Scaleway Developer documentation](https://www.scaleway.com/docs/serverless/messaging/) provides clear information on how this managed service works and can be configured.
```

Comment:
Link to "developer documentation" is wrong, it should point to developer website. Or change the text of the link to "Scaleway documentation" and the link to /serverless/messaging/concepts/ (or QS?)